src/pages/index.html

```diff
@@ -250,7 +250,8 @@ <h5>Tan et al.</h5><span>DOI:10.1101/2021.10.10.463703</span></a>
                 <div class="column is-4">
                     <figure class="testimonial">
                         <blockquote>
-                            Here, we present OnClass, an algorithm and accompanying software for automatically classifying cells into cell types that are part of the controlled vocabulary that forms the Cell Ontology.
+                            Here, we present OnClass, an algorithm and accompanying software for automatically classifying cells into cell 
+                            s that are part of the controlled vocabulary that forms the Cell Ontology.
```

Comment:
is there a reason for this line break and change? 
I'm guessing this is a mistake - in which case can I change it back?

Reply:
Editing mistake (was working on it on the train). Please correct.  Thanks for spotting !